radixdlt/src/main/java/com/radixdlt/consensus/sync/SyncedRadixEngine.java

```diff
@@ -90,7 +92,7 @@ public SyncedRadixEngine(
 		StateSyncNetwork stateSyncNetwork
 	) {
 		if (epochChangeView.isGenesis()) {
-			throw new IllegalArgumentException("Epoch change view must not be genesis.");
+			Panicker.panic("Epoch change view must not be genesis.");
```

Comment:
Not sure if should be using Panicker within these modules as it breaks unit testing. I suggest we use panicker only from outside the modules.

Reply:
Panicker has support for unit testing.

Reply:
Ah, gotcha. Though it still seems to me to break encapsulation? What's the win with using Panicker?